docker-compose.yml

```
@@ -0,0 +1,39 @@
+version: '3.8'
+services:
+  app:
+    container_name: task-api
+    build: 
+      dockerfile: Dockerfile
+      context: .
+    ports:
+      - "8090:8090"
+    depends_on:
+      db:
+        condition: service_healthy
+    environment:
+      MYSQL_USER: root
+      MYSQL_PASSWORD: ${MYSQL_ROOT_PASSWORD:-your_root_password}
+      # https://stackoverflow.com/questions/24319662/from-inside-of-a-docker-container-how-do-i-connect-to-the-localhost-of-the-mach
+      MYSQL_HOST: host.docker.internal
+      MYSQL_PORT: 3308
+      MYSQL_DATABASE: sample
+  db:
+    container_name: mysql
+    image: mysql:latest
```

Avoid using latest whenever possible. Many say it makes it difficult tracking what version of the mysql image is being used.
Here's [mysql image tags](https://hub.docker.com/_/mysql)
 
```suggestion
    image: mysql:8.033
```